### PR TITLE
Add some too bloated `Math` polyfills

### PR DESCRIPTION
**Pull request by zloirock**
Resume #39

### CODE REVIEW

File: manifests/native.json [hunk 1]
```diff
@@ -248,6 +248,134 @@
...
+      "type": "native",
+      "moduleName": "math.log1p",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.log1p",
+      "mdnPath": "Global_Objects/Math/log1p",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.log2",
```
Comment (43081j) [inline manifests/native.json:357]: this one doesn't seem to exist on npm 👀 

can you double check it before i merge? 🙏

File: manifests/native.json [hunk 2]
```diff
@@ -248,6 +248,134 @@
...
+      "type": "native",
+      "moduleName": "math.acosh",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.acosh",
+      "mdnPath": "Global_Objects/Math/acosh",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.asinh",
```
Comment (43081j) [inline manifests/native.json:261]: this one also doesn't seem to be on npm

File: manifests/native.json [hunk 3]
```diff
@@ -248,6 +248,134 @@
...
+      "type": "native",
+      "moduleName": "math.fround",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.fround",
+      "mdnPath": "Global_Objects/Math/fround",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.hypot",
```
Comment (43081j) [inline manifests/native.json:325]: this one too

File: manifests/native.json (hunk 1 above)
Comment (43081j) [reply manifests/native.json:357]: the ones that don't seem to exist (anymore?):

- hypot
- expm1
- cosh
- asinh
- log2

File: manifests/native.json (hunk 2 above)
Comment (zloirock) [reply manifests/native.json:261]: The same, https://github.com/es-shims/Math.asinh

File: manifests/native.json (hunk 3 above)
Comment (zloirock) [reply manifests/native.json:325]: The same, https://github.com/es-shims/Math. hypot

File: manifests/native.json (hunk 1 above)
Comment (zloirock) [reply manifests/native.json:357]: Interesting, see https://github.com/es-shims/Math.log2

Comment (43081j) [reply manifests/native.json:357]: i suppose they never got published

we could leave them out and re-introduce them if they ever get published possibly?

as someone could claim the name meanwhile with a lighter polyfill

Comment (zloirock) [reply manifests/native.json:357]: Sure.

**Review by 43081j — APPROVED** [review]

**Pull request merged**